agda/Structure/Bin.agda

```diff
@@ -31,8 +31,8 @@ module Structure.Bin
 
   data DBin-g (A : Set) : ℕ → Set where
     ϵ : ∀ {n} → DBin-g A n
-    _O : ∀ {n} → (bs : DBin-g A (suc n)) → DBin-g A n
-    _[_]I : ∀{n} → (bs : DBin-g A (suc n))(t : T A n) → DBin-g A n
+    _O : ∀ {n} (bs : DBin-g A (suc n)) → DBin-g A n
+    _[_]I : ∀ {n} (bs : DBin-g A (suc n)) (t : T A n) → DBin-g A n
```

Comment:
Stylistic change, postpone to another PR performing a global and uniform  update.